ONSdigital/eq-survey-runner
# eq-247 Census Theme Framework

### What is the context of this PR?

[Trello card](https://trello.com/c/tD19yTHy/247-3-census-theme-english)

This PR implements the basic framework for the Census theme:
1. Census `page-header` (this has been renamed from `survey-header`).
2. Census footer.
3. Census brand colour.
4. Census landing page template (as per prototype)

This PR does not implement:
1. Mobile/responsive styles.
2. Any census content (progressing past landing page will throw an error)
### How to review
1. Start the `census_theme` survey.
2. Confirm correct theme is chosen.

![screenshot 2016-10-21 16 26 46](https://cloud.githubusercontent.com/assets/930398/19603983/3d5a0cc2-97ab-11e6-9c4f-629c74327b17.png)
